src/TimeGridHeader.js

```diff
@@ -47,7 +47,8 @@ class TimeGridHeader extends React.Component {
       getDrilldownView,
       getNow,
       getters: { dayProp },
-      components: { header: HeaderComponent = Header },
```

Comment:
You should leave header from lib as default if header not came from outside

Reply:
updated